### PR TITLE
Merge Of Develop Into Master

### PR DESCRIPTION
**Pull request by RichmarIII**

### CODE REVIEW

**Comment by Richmar1** [comment]
Review

**Review by RichmarIII — COMMENTED** [review]
Approved

File: TargaImage.h [hunk 1]
```diff
@@ -4,6 +4,25 @@
 #include <stdint.h>
 #include <memory>
 
+
+
+#ifndef FASTTARGA_HEADERONLY
```
Comment (RichmarIII) [inline TargaImage.h:9]: Looks Good

**Review by RichmarIII — COMMENTED** [review]
Approved

**Pull request merged**